Assets/Prefabs/Systems/AwakenessManager.prefab

```diff
@@ -11,7 +11,7 @@ GameObject:
   - component: {fileID: 2744821312385824694}
   - component: {fileID: 2744821312385824693}
   m_Layer: 0
-  m_Name: GameManager
+  m_Name: AwakenessManager
```

Comment:
Wait does this mean you deleted the GameManager prefab and made it the AwakenessManager?

If so, we'll probably still need a GameManager prefab at some point.

Reply:
Yup, that's exactly what I did.  And yeah, this means that we can add a GameManager prefab to handle more general game-related tasks, and the AwakenessManager exclusively handles the awakeness mechanic.